src/charm.py

```diff
@@ -190,7 +190,7 @@ def _oathkeeper_service_is_running(self) -> bool:
     @property
     def _forward_auth_config(self) -> ForwardAuthConfig:
         return ForwardAuthConfig(
-            decisions_address=f"https://{self.app.name}.{self.model.name}.svc.cluster.local:{OATHKEEPER_API_PORT}/decisions",
+            decisions_address=f"http://{self.app.name}.{self.model.name}.svc.cluster.local:{OATHKEEPER_API_PORT}/decisions",
```

Comment:
shouldn't this be a variable?
can't see traefik guys being happy long term with http only